### PR TITLE
Migrate Google importers to the new executor API

### PR DESCRIPTION
**Pull request by xokker**
Following #1079 migrating Google photo and video importers to the new API. The new API supports size reporting.

### CODE REVIEW

File: extensions/data-transfer/portability-data-transfer-google/src/main/java/org/datatransferproject/datatransfer/google/photos/GooglePhotosImporter.java [hunk 1]
```diff
@@ -297,32 +294,32 @@ long importPhotoBatch(
...
-          title,
-          () -> {
-            throw new IOException(
-                String.format(
-                    "Media item could not be created. Code: %d Message: %s",
-                    status.getCode(), status.getMessage()));
-          });
+      executor.importAndSwallowIOExceptions(
+          item,
+          inp ->
```
Comment (jzacsh) [inline extensions/data-transfer/portability-data-transfer-google/src/main/java/org/datatransferproject/datatransfer/google/photos/GooglePhotosImporter.java:310]: nit: I'm not sure what `inp` is an acronym for. Could you give a more descriptive name to `inp` variable - both here and elsewhere like line 305? eg: maybe `itemToImport` or something, assuming I'm understanding correctly.

**Comment by jzacsh** [comment]
LGTM (other than my tiny comment).

**Review by jzacsh — APPROVED** [review]

**Review by wmorland — APPROVED** [review]
Comments are addressed

**Comment by wmorland** [comment]
@jzacsh Your review seems to be blocking and unfortunately I don't see the option to dismiss https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/dismissing-a-pull-request-review We're good to merge this if you have no other questions.

**Review by jzacsh — APPROVED** [review]

**Pull request merged**